settings.yml

```diff
@@ -3,6 +3,26 @@ variables:
   ################################################## Common Variables #######################################################
   ###########################################################################################################################
 
+  # Enable or disable CARMLs CI telemetry tracking. Note: This tracks the number of deployments only.
+  # See: https://github.com/Azure/ResourceModules/wiki/The%20library%20-%20Module%20design#telemetry
+  enableDefaultTelemetry: true
+
+  ######################################
+  # Local tokens settings
+  ######################################
+
+  # the 'localToken_' prefix will be removed from the key name when the pipelines run.
+  # e.g. if you have a token in your parameter file as <<customKey>>, then the token defined in this file looks like "localToken_customKey": 'value'
+  localToken_namePrefix: 'carml' # A 3-5 character length string, included in the resources names, is overridden if provided as GitHub Secret / ADO Variable
```

Comment:
I guess we'll leave this empty?

Reply:
see other comment that describes the different options. I was thinking of renaming it to `cntso` instead if we set this as a placeholder for customers.

Reply:
@MrMCake  @eriqua  Updated and set to empty as per option (2)